Winium/Winium.Mobile.Driver/CommandExecutors/CloseAppExecutor.cs

```diff
@@ -12,9 +13,16 @@ internal class CloseAppExecutor : CommandExecutorBase
 
         public static void CloseApp(Automator automator)
         {
-            var remoteCommand = new Command(DriverCommand.CloseApp);
-            automator.CommandForwarder.ForwardCommand(remoteCommand);
-            Thread.Sleep(TimeSpan.FromMilliseconds(500));
+            if (automator.Deployer.AppType == AppType.XAP)
```

Comment:
Maybe we should add `IsTerminatable` into `Deployer`? At the moment this condition is basically duplicated both her and in Deployer.
Or we can make `Terminate` return a bool: `true` if it can and did in fact terminated the app, `false` if it can not.